src/main/java/edu/greenblitz/robotName/utils/PIDObject.java

```diff
@@ -0,0 +1,192 @@
+package edu.greenblitz.robotName.utils;
+
+import edu.wpi.first.math.controller.PIDController;
+
+public class PIDObject {
+
+    private double kp, kd, ki, ff;
+
+    private double iZone;
+
+    private double tolerance;
+
+    private int inverted = 1;
```

Comment:
maybe change to boolean

Reply:
that is a code design. if we change it to boolean than we better rewrite this shitty code from the beginning